src/averageAges.js

```diff
@@ -1,61 +1,64 @@
 'use strict';
 
-/**
- * Implement calculateMenAverageAge function
- *
- * Function returns average age of men in array. If `century` is specified then
- * function calculates average age only for men who died in this century
- *
- * To calculate century:
- * Divide year of person's death by 100: Math.ceil(person.died / 100)
- *
- * @param {object[]} people
- * @param {number} century - optional
- *
- * @return {number}
- */
+function getAverageAge(people) {
+  return people.reduce((sum, person) => (
+    sum + person.died - person.born
+  ), 0) / people.length;
+}
+
+function getPeopleBySex(people, sex) {
+  return people.filter(person => person.sex === sex);
+}
+
+function getPersonHasChild(woman, people) {
+  return people.some(person => person.mother === woman.name);
+}
+
+function calculateAgeDiffrens(people, womenHaveChildrens) {
+  const ageDiff = people.map(person => {
+    const personMother = womenHaveChildrens.find(woman =>
+      person.mother === woman.name
+    );
+
+    return (!personMother) ? false : person.born - personMother.born;
+  }).filter(mother => mother);
+
```

Comment:
Ланцюжек методів було б непогано рознести на різні рядки, та й тернарний вираз теж)
```suggestion
  const ageDiff = people
    .map(person => {
      const personMother = womenHaveChildrens.find(woman =>
        person.mother === woman.name
      );

      const presentInList = !personMother;

      return (presentInList) 
        ? false
        : person.born - personMother.born;
    })
    .filter(mother => mother);

```